QEfficient/cloud/compile.py

```diff
@@ -120,7 +133,8 @@ def main(
         help="Cloud AI 100 device ids (comma-separated) e.g. [0] ",
     )
     parser.add_argument(
-        "--aic_enable_depth_first", "--aic-enable-depth-first",
+        "--aic_enable_depth_first",
+        "--aic-enable-depth-first",
```

Comment:
duplicate

Reply:
Coming from pre-commit linter.

Reply:
ISnt that coming because in line 123 of original file, this is mentioned twice?

  "--aic_enable_depth_first", "--aic-enable-depth-first",

Reply:
No, it's coming due to pre-commit linter.